libs/widget-lib/docs/README.md

```diff
@@ -75,99 +74,57 @@ An example of connecting a widget to Metamask:
-| `dynamicHeightEnabled` | `boolean`              | (Optional) Dynamically changes the height of the iframe depending on the content.                                                                            |
-| `enabledTradeTypes`    | `Array<TradeType>`     | (Optional) CowSwap provides three trading widgets: swap, limit and twap orders. Using this option you can narrow down the list of available trading widgets. |
-| `palette`              | `CowSwapWidgetPalette` | (Optional) Using the palette you can customize the appearance of the widget. For example, you can change the main color of the background and text.          |
-
-```typescript
-export interface CowSwapWidgetMetaData {
-  appKey: string
-  url: string
-}
-
-interface TradeAsset {
-  asset: string
-  amount?: string
-}
-
-export interface TradeAssets {
-  sell: TradeAsset
-  buy: TradeAsset
-}
-
-export enum TradeType {
-  SWAP = 'swap',
-  LIMIT = 'limit',
-  ADVANCED = 'advanced',
-}
-
-export interface CowSwapWidgetPalette {
-  primaryColor: string
-  screenBackground: string
-  widgetBackground: string
-  textColor: string
-}
-```
+>All params are optional
+
+| Parameter              | Type                    | Description                                                                                                                                                  |
+|------------------------|-------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `width`                | `number`                | The width of the widget in pixels. Default: 400px                                                                                                            |
+| `height`               | `number`                | The height of the widget in pixels. Default: 600px                                                                                                           |
+| `metaData`             | `CowSwapWidgetMetaData` | Information about the application in which the widget is embedded. This information will help identify the source of orders and requests from users.         |
+| `provider`             | `EthereumProvider`      | The Ethereum provider to be used for interacting with a wallet.                                                                                              |
+| `chainId`              | `number`                | The blockchain ID on which the trade will take place.                                                                                                        |
+| `tradeType`            | `string`                | The type of trade. Can be `swap` or `limit-orders`.                                                                                                          |
```

Comment:
Fixed. Also changed type from string to TradeType.
Since the lib is only available as a typescript code, there are no chance to choose something wrong, only what `TradeType` enum allows